packaging/hudi-cli-bundle/pom.xml

```diff
@@ -0,0 +1,244 @@
+                <transformer implementation="org.apache.maven.plugins.shade.resource.ApacheNoticeResourceTransformer">
+                  <addHeader>true</addHeader>
+                </transformer>
+                <transformer implementation="org.apache.maven.plugins.shade.resource.IncludeResourceTransformer">
+                  <resource>META-INF/LICENSE</resource>
+                  <file>target/classes/META-INF/LICENSE</file>
+                </transformer>
+                <transformer implementation="org.apache.maven.plugins.shade.resource.AppendingTransformer">
+                  <resource>META-INF/spring.handlers</resource>
+                </transformer>
+                <transformer implementation="org.apache.maven.plugins.shade.resource.AppendingTransformer">
+                  <resource>META-INF/spring.schemas</resource>
+                </transformer>
+                <transformer implementation="org.apache.maven.plugins.shade.resource.AppendingTransformer">
+                  <resource>META-INF/spring.factories</resource>
+                </transformer>
+                <transformer implementation="org.apache.maven.plugins.shade.resource.ServicesResourceTransformer"/>
+                <transformer implementation="org.apache.maven.plugins.shade.resource.ManifestResourceTransformer">
+                  <mainClass>org.apache.hudi.cli.Main</mainClass>
+                </transformer>
+                <transformer implementation="org.apache.maven.plugins.shade.resource.AppendingTransformer">
+                  <resource>META-INF/spring/org.springframework.boot.actuate.autoconfigure.web.ManagementContextConfiguration.imports</resource>
+                </transformer>
+                <transformer implementation="org.apache.maven.plugins.shade.resource.AppendingTransformer">
+                  <resource>META-INF/spring/org.springframework.boot.autoconfigure.AutoConfiguration.imports</resource>
+                </transformer>
+              </transformers>
+              <artifactSet>
+                <includes combine.children="append">
+                  <!-- hudi -->
+                  <include>org.apache.hudi:hudi-cli</include>
+                  <include>org.apache.hudi:hudi-utilities_${scala.binary.version}</include>
+                  <!-- cli related dependencies -->
+                  <include>com.fasterxml:classmate</include>
+                  <include>com.fasterxml.woodstox:woodstox-core</include>
+                  <include>com.google.code.gson:gson</include>
+                  <include>com.google.re2j:re2j</include>
+                  <include>com.jakewharton.fliptables:fliptables</include>
+
+                  <include>jakarta.el:jakarta.el-api</include>
+                  <include>jakarta.validation:jakarta.validation-api</include>
+                  <include>net.java.dev.jna:jna</include>
+
+                  <include>org.apache.commons:commons-configuration2</include>
```

Comment:
As the `commons-configuration2` dependency is removed, this `include` entry is no longer needed.

Reply:
I resolved it.